var/spack/repos/builtin/packages/darshan-util/package.py

```diff
@@ -18,7 +18,10 @@ class DarshanUtil(Package):
 
     maintainers = ['shanedsnyder', 'carns']
 
-    version('develop', branch='master')
+    version('master', branch='master', submodules='True')
+    version('3.3.0',      sha256='2e8bccf28acfa9f9394f2084ec18122c66e45d966087fa2e533928e824fcb57a', preferred=True)
+    version('3.3.0-pre2', sha256='0fc09f86f935132b7b05df981b05cdb3796a1ea02c7acd1905323691df65e761')
+    version('3.3.0-pre1', sha256='1c655359455b5122921091bab9961491be58a5f0158f073d09fe8cc772bd0812')
```

Comment:
same
```suggestion
```

Reply:
I would have suggested the same in upstream review. But I agree that it's harmless to keep it.